packages/react-native-reanimated/Common/cpp/worklets/WorkletRuntime/ReanimatedHermesRuntime.cpp

```diff
@@ -5,19 +5,14 @@
 #if JS_RUNTIME_HERMES
 
 #include <cxxreact/MessageQueueThread.h>
+#include <hermes/hermes.h>
 #include <jsi/decorator.h>
 #include <jsi/jsi.h>
 
 #include <memory>
 #include <string>
 #include <utility>
 
-#if __has_include(<reacthermes/HermesExecutorFactory.h>)
-#include <reacthermes/HermesExecutorFactory.h>
```

Comment:
Can you please confirm in which version of RN `HermesExecutorFactory.h` file was removed?

Reply:
Actually, both files exist, but `#include <hermes/hermes.h>` is all we need to use. I've run compatibility tests on CI, and this change appears to be safe (expect one flaky check)